code/modules/research/designs/autolathe_desings/autolathe_designs_tcomms_and_misc.dm

```diff
@@ -271,4 +271,13 @@
 	build_type = AUTOLATHE
 	materials = list(/datum/material/iron = 5000, /datum/material/glass = 2000)
 	build_path = /obj/item/vending_refill/custom
-	category = list("initial", "Misc")
\ No newline at end of file
+	category = list("initial", "Misc")
+
+/datum/design/trigger_assembly
+	name = "Trigger Assembly"
+	id = "trigger_assembly"
+	build_type = AUTOLATHE
+	materials = list(/datum/material/iron = 10500, /datum/material/glass = 50)
```

Comment:
Um. 5 sheets metal? 6500 (around 3.25 sheets) would be a bit more reasonable for a *trigger*.

Reply:
Changed to this, the 0.25 iron sheets and glass is from the igniter that you'd use to otherwise craft it manually.